rholang/src/main/scala/coop/rchain/rholang/interpreter/matcher/package.scala

```diff
@@ -24,37 +28,38 @@ package object matcher {
   type NonDetFreeMapWithCost[A] = StateT[StreamWithCost, FreeMap, A]
   type StreamWithCost[A]        = StreamT[ErroredOrCostA, A]
 
+  type _freeMap[F[_]] = MonadState[F, FreeMap]
```

Comment:
it's a bit unusual for the type to be lowercased, i was confused when I saw the usage

Reply:
you'r saying this is convention?

Reply:
See my reply to Łukasz. It's up to us to shape our conventions. I've seen it in the wild though, in the `eff` lib's docs. I guess I'll add my explanation to a comment near those type aliases :)